flow/core/kernel/vehicle/traci.py

```diff
@@ -536,13 +556,6 @@ def get_num_not_departed(self):
         """See parent class."""
         return self.num_not_departed
 
-    def get_fuel_consumption(self, veh_id, error=-1001):
-        """Return fuel consumption in gallons/s."""
-        ml_to_gallons = 0.000264172
-        if isinstance(veh_id, (list, np.ndarray)):
-            return [self.get_fuel_consumption(vehID, error) for vehID in veh_id]
-        return self.__sumo_obs.get(veh_id, {}).get(tc.VAR_FUELCONSUMPTION, error) * ml_to_gallons
-
```

Comment:
if we strip this out, we need to know what to plug in in its place. @Yasharzf and/or Sulaiman can help